app/src/main/java/com/example/tasky/login/domain/LoginViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.tasky.login.domain
+
+import androidx.compose.ui.text.input.TextFieldValue
+
+class LoginViewModel {
+}
+
+data class LoginState(
```

Comment:
I like to have every parameter in a state, with default values
Mostly because when the screen renders for the first time, it should have an initial state